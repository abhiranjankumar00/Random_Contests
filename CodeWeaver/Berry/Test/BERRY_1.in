18
42 870
5503 32458 14671 3969 220 359 12449 17488 18783 19785 19962 5514 21793 14838 11605 1936 19979 25734 26366 17102 9961 28770 23878 31389 30400 17740 4731 15208 7778 28320 6935 25614 27711 32729 32824 20104 12926 8790 9808 27596 845 12383 
305 78
22257 1909 355 30173 9107 9009 19331 22715 27513 23872 15957 6796 24437 15417 15073 31168 24460 3615 19696 12690 24151 20021 18823 11907 5033 7443 13998 26375 17011 32506 24693 11390 5604 21605 16185 2149 22996 16608 4900 31182 4706 29725 22771 9997 14044 2373 31740 22453 5088 28812 26991 19139 6337 5896 26844 15640 5164 16579 24053 13357 9228 18703 22422 24834 23722 15641 4098 12119 27417 1217 17008 21791 14033 3497 31174 30258 18074 11404 15524 12354 27820 10450 15012 8976 32276 9825 24288 18655 6489 25013 27573 13097 16480 29235 967 32658 18829 1722 17477 6426 27691 20604 21615 6550 27662 4108 3669 24417 10358 30903 9441 11087 4663 24088 27415 23266 19735 24551 8348 4801 120 2066 26485 28005 6967 3855 18194 534 3795 14960 24715 22550 17874 2488 14377 6684 22880 9581 14376 7683 19002 17518 20667 5316 16586 31623 22865 30370 6291 11075 5911 32676 15056 32769 3262 20552 3160 14410 30590 1654 29381 9570 7515 25267 13525 6685 20647 19863 14865 15348 19656 20865 28076 27224 20539 23689 18605 12359 6105 24246 18257 29724 8025 6258 19882 22955 19223 11578 16269 2701 24339 20122 20395 22713 26429 4953 18942 28500 29936 20209 23911 1483 21948 32065 10389 18718 10088 5766 3624 28543 27959 24456 5142 10779 2667 2577 21070 26936 17928 14755 13468 9856 15322 16490 5069 10652 24249 10352 27155 31493 28684 23824 9899 30999 4236 2221 25788 17256 20043 31396 2435 28759 21492 10622 3501 16616 7508 9579 30212 18127 21785 3820 16206 12490 16346 26063 16754 12596 22616 17504 19933 13016 260 23262 3364 20483 28353 16172 24555 16349 12522 25801 18181 11768 31383 20738 5853 12330 4380 24422 31252 20120 979 10875 1899 21012 4380 27823 28388 19625 23713 28049 548 4211 23263 20289 7196 2228 5602 20517 11240 10533 12111 21726 26359 
133 482
17320 20091 26221 29577 4812 20716 13253 8380 4541 28089 2235 14085 18854 9972 18025 7458 10269 3692 26544 4481 9381 25891 29401 25941 24439 20226 11900 28137 7554 28364 7585 8244 17840 32337 1830 2735 17259 14052 3169 8547 29280 7694 4869 4166 30594 2692 1610 1991 11090 30039 13128 14248 31070 27499 17572 27660 22792 13098 8559 209 17289 31353 13131 7967 19254 8427 22480 31041 14337 29237 302 30900 19778 25827 18963 4734 7352 12617 207 13761 2762 21691 28086 2192 26643 21761 22725 26369 17438 22533 4745 22660 23918 25551 1085 28531 21186 23219 2867 18154 31127 1993 9077 4824 32237 20382 9643 30294 12110 22825 7231 5176 7949 17153 29632 3554 29644 14541 2692 25694 5696 31995 25490 28587 6969 15029 190 24663 3804 13328 10262 32592 1331 
364 303
6269 8183 5097 20393 29078 30838 6478 25614 21220 21587 29857 14991 30255 21830 5007 20918 18729 13896 30967 17780 19025 17645 8432 13074 11544 1267 26125 6506 2370 12827 19424 2391 6544 5175 21180 14954 19868 22917 14527 22495 13060 27451 19472 6607 31178 28771 12902 32423 6139 29417 18890 14552 20623 23283 1693 9424 8593 13424 29404 23338 23936 29428 12963 13089 29684 10179 12784 18763 11652 24108 24490 24196 24296 4632 6626 8999 22363 29922 12120 17029 3651 29801 6721 17039 21524 14436 22599 3030 2674 2550 978 11702 10134 22915 4742 13005 2290 22209 23821 6578 22808 20242 21526 17892 3288 17937 1693 32001 15272 31850 23917 17465 22346 22768 12260 12801 1704 26601 5623 2060 10243 25772 7029 10615 15948 367 14480 16708 19922 24922 13209 11529 27678 30944 20491 32745 1819 18510 28363 12740 10107 9380 942 20139 12885 677 1084 14999 28179 30762 13236 23898 20107 26555 28752 19157 19564 2656 26057 15212 19420 19381 18718 26807 22111 11325 402 8826 11259 7672 25331 12248 28570 3896 23842 20675 29359 6064 17616 29623 25628 31694 6534 29608 26196 31307 27880 29241 20668 6144 20282 8750 8280 24059 25891 5668 23459 15826 2737 26495 28094 4151 10142 18853 15565 25037 6354 23914 32671 570 21288 22730 5773 2430 9077 22238 27556 18307 12231 25609 7686 20980 7658 6771 31885 9299 817 25272 5042 1606 370 11489 21165 11314 13651 13715 3038 17931 22980 11142 21612 28779 17613 18745 1836 15329 8586 14052 28356 16011 2932 18607 23312 25575 28385 27937 9668 28390 21199 24539 27219 25154 28637 29830 29968 17170 14490 3594 11667 27036 14082 5632 95 21610 25414 2155 3010 12704 22476 26530 5116 4748 1655 11409 675 32127 21288 1825 30021 20955 14213 757 8016 12910 21497 25687 815 27134 4603 20850 18102 32293 15252 7105 9920 25696 11291 15815 19990 3426 32324 24833 5011 15022 23385 32793 25478 21092 20422 31068 22616 9563 18651 9582 18031 23409 8142 17980 16209 31263 22015 25139 20493 14673 26240 24496 32471 6772 20693 29879 19442 30160 16632 16103 14803 29208 30881 6061 8323 6719 24003 30905 20549 1422 21082 1198 18297 17908 14692 2078 32704 4253 19757 1717 
121 705
21701 10960 10420 21483 13519 14075 7329 22300 5521 16370 16701 26370 14323 215 11191 12384 4280 27176 24095 29267 21147 21385 16925 24117 24222 31428 15331 11970 3743 29710 26976 14969 3628 28556 25015 1349 13720 30741 2287 5469 6990 3898 19436 3945 20326 19075 22686 24038 30070 22012 9848 26571 12459 32752 25380 6765 5656 12789 6005 19104 6477 31528 6301 12575 10028 4026 6560 1582 25336 25004 28936 125 14767 14038 26331 15184 16282 24622 7882 18397 3106 30279 29355 28149 2021 16152 20777 24551 24841 8447 29882 26018 6608 18182 1746 17177 25965 23140 855 24044 18199 30023 28756 26180 10075 13008 15857 1790 21430 95 25251 24845 7267 5138 1952 22041 1138 11400 22934 26220 14362 
220 211
20892 9743 15696 28717 2665 3376 4760 4753 30147 10183 12316 26734 1595 26746 7931 29488 8472 8893 6883 7583 27793 28733 29154 27748 20031 1407 5753 6088 11729 14788 6131 29376 20482 17969 29940 17191 24008 32812 31829 28490 27598 4873 12335 9385 5669 31974 24374 23548 1079 21203 26697 24181 26876 4151 23623 12357 21360 30063 29219 1121 25412 14775 315 7558 13779 5198 30181 16506 8025 24789 29771 7062 1119 25336 7546 18305 26490 27985 10933 17726 32565 28553 21263 7529 26700 12225 12089 1213 18459 13104 3992 10714 29525 6669 30874 4165 27897 29359 14667 7345 20866 18419 20515 13949 607 28382 4642 8829 9634 22403 18464 31485 19964 25895 13918 26883 24297 4516 16992 725 296 24587 11007 29627 15214 25229 1722 20742 859 22428 31821 16465 23213 8781 27013 21255 19636 8705 27730 15142 24582 11588 542 15682 20595 13301 12637 972 9531 19624 29029 14228 30591 12616 17536 397 31990 28417 14400 22992 10977 19804 16403 18404 1345 12460 7703 30781 28231 31658 20016 19202 2572 13404 10071 16404 2690 28731 10037 25675 30635 19348 7152 9219 18440 28719 8261 9943 23893 28463 7639 6316 14707 29241 18886 1010 32678 2008 20356 8486 5632 3872 7652 16283 31517 1682 2676 11131 9233 16960 16141 13576 30367 19762 9640 15656 13228 31239 18035 27425 
404 74
26335 19493 28577 10489 30841 8846 30977 23619 4249 25458 15562 5831 941 1431 14969 28322 4527 541 7039 26888 26189 25614 21644 9856 25672 30262 27661 8017 16410 821 2548 1079 11739 23506 13495 5979 9829 6034 24475 4482 1975 17290 11826 26501 5271 29553 4386 1025 6012 26873 8233 18767 11434 17759 21854 28599 30623 31041 3514 5088 18350 26289 29398 2443 28650 27015 12790 18049 22085 14843 9287 31178 32249 23923 19557 8992 4391 6553 19744 16036 11710 7601 15744 22735 31135 22058 32263 7850 16895 7794 29493 15938 10764 17679 18770 11094 11475 5612 9575 7252 30305 24304 26510 21380 22568 8917 25195 2178 23717 28216 11259 1521 15936 748 27532 20334 17780 19860 18701 25539 21039 22897 24968 28509 5244 13944 25837 769 14431 1448 16657 8890 2304 8090 32246 16662 20236 2394 12109 31377 28249 11125 7993 9554 1737 32595 5718 2325 7280 9927 25850 21353 2809 8677 4539 7195 18501 5908 20785 3570 14934 24932 11005 1948 9324 22817 28681 18665 28525 2728 26130 32823 20874 20345 19556 19502 6432 32142 7653 1453 7900 8427 13397 26115 8995 29559 12500 23907 6833 1802 19877 11666 11904 21959 32049 7395 29419 11436 31311 21861 6675 9319 11714 17499 9602 7275 3331 3564 23310 27716 22082 26908 256 16167 19879 30715 9590 19918 24541 28700 29958 27267 19921 10057 25764 4986 17847 22645 12998 32611 3407 13554 966 22592 8550 5605 7559 6260 28319 25078 1627 15901 1907 31564 2852 18596 31607 18872 28858 13459 13277 13616 21645 27046 31038 9344 140 20260 1687 21564 13893 31343 19857 6649 13645 11226 20485 26556 226 3516 12991 9139 10447 27075 28034 10275 6544 18570 15437 32674 14263 12141 13789 12678 19086 28828 11123 12957 31230 16750 13783 19999 25519 2808 5021 6880 2929 10463 15527 20682 10971 2666 20203 4747 9265 27099 14454 32651 7307 3584 17073 8737 308 18949 25316 3590 31825 12172 29688 17162 7363 9983 15745 10245 13646 25125 27644 12817 14074 23796 24148 3062 2745 24743 27820 20966 11851 15632 3160 13675 6239 11310 29996 7581 10235 5122 11258 6040 8989 6815 5718 11053 2211 16513 31644 26584 14696 29983 5941 15858 15536 22979 8213 30760 9158 9882 27016 26924 20707 26119 303 8618 9554 31293 28229 17022 23250 8461 30247 16164 3132 27132 2580 9328 12645 21145 16945 14207 4678 32014 2512 237 30042 13556 24817 6216 3400 13932 22281 17349 27074 27499 8963 16434 
121 565
4949 31877 17708 7298 2254 6772 6546 6388 6605 31418 19514 24275 9713 22343 25826 30256 30489 27733 8553 3522 2095 29681 4927 29320 11844 31415 12570 10928 22498 29149 12522 14264 22173 8819 15888 17363 26348 26088 24522 16014 27191 18385 9202 11443 1841 29926 25065 12141 9320 6989 10702 1710 28955 8220 13299 4814 28747 19993 25745 6517 14868 25028 14266 20922 26430 5783 10640 31628 23312 6540 28341 1617 24420 1248 4354 2766 18177 18710 17532 7239 2596 10048 2179 13543 4448 8314 26957 16738 8872 32439 30099 4056 9387 23232 15498 9725 11360 17273 26645 17015 2273 27238 18233 3463 16764 31087 23761 15596 14855 30176 18051 12036 29045 21684 4082 16693 3751 9235 17973 25995 12164 
383 161
24457 15266 11852 14553 19266 12487 20777 18338 15880 27482 6152 385 3647 1398 7334 8454 13511 23253 14574 4427 17894 28141 26498 7219 30338 10335 4760 19952 404 31378 17671 12744 473 7835 29089 19480 5067 609 17604 30105 21455 7422 13356 31714 3248 13160 16651 11054 10828 20560 8284 9568 17549 29514 15732 10820 22171 15151 19162 13592 30288 4031 21848 4939 8173 3723 3410 22660 27147 16147 14935 21478 13780 21035 3318 27283 12146 28835 17107 31598 13000 23846 20730 12326 26720 28003 2162 24432 11941 7787 26902 25747 9043 18522 5792 4138 24875 13626 9223 5669 17899 7972 10507 7442 21629 22952 22029 21147 1903 10864 31269 10575 10147 5407 12143 9125 31560 7807 8613 20541 24840 19164 8947 23402 11139 23467 774 23022 20733 4208 23655 12548 17235 28382 19463 16292 1076 22079 18203 11522 18829 25110 809 88 17989 24579 9315 26085 27435 23784 9781 17717 13357 3402 12826 3236 21962 5370 22707 22046 24266 29172 24858 18728 8748 3719 8820 24100 32096 16054 7109 26320 20150 11487 15881 32785 12311 31130 7296 20719 18931 4836 30537 15072 21114 1661 21554 24393 3343 21390 6607 7746 24057 32655 24777 29338 18012 29288 28537 20250 23656 24022 5045 24846 5073 13329 20202 23554 27263 29100 2155 13002 19846 16060 14857 25029 19032 11068 19172 11874 24634 2736 3201 32738 1524 13065 3612 13664 14285 8905 14911 7439 8630 21095 29331 28868 14790 13557 7671 31668 24294 11264 23759 19838 20430 29368 22430 7788 3632 17488 23512 18677 562 16808 15089 31879 29218 23082 8122 3460 8805 15346 19949 1675 12721 3889 32774 24312 1405 211 22357 31406 23221 24671 4690 22292 20145 21791 32048 2397 29800 28290 20661 29197 18913 5054 29512 18872 8683 5817 20556 27405 22030 28202 14764 32276 21697 23291 1975 26804 24541 31987 27439 10360 3922 6801 9628 31123 27673 8251 7142 28449 14186 6808 30499 24751 12846 12346 18350 20734 30903 24192 24185 12804 18095 32186 20644 10804 14158 20623 22862 16127 21968 8860 3499 2203 4647 14942 5974 21251 28141 8786 26857 20543 30108 3418 8396 16357 23041 23139 3658 12256 15854 31557 3306 14960 3120 17130 10748 25970 27072 24243 18546 1762 6206 32535 14065 21150 23706 29582 27688 30060 15893 15789 23905 24895 12648 24466 9045 11958 1090 27010 24901 
100 286
7416 2769 8774 20569 25208 10754 8413 15958 24704 18480 11467 22883 31757 13229 9002 29193 19477 19944 11449 26327 27256 26772 13941 2730 20789 32262 22633 16427 6105 18655 7878 28312 9534 32492 26934 13256 10609 13130 21614 30569 29684 9166 25090 17293 31270 30115 21118 1637 8703 9525 26034 8523 9472 11598 30029 26886 8042 10623 7598 31562 8111 30658 26870 3455 18982 7517 12386 6339 25858 29450 10200 1842 24709 23636 18367 18021 12145 3652 2324 25400 960 10557 10170 4817 17300 10789 24338 19678 19057 30405 21708 23325 19114 2419 9725 546 25369 32750 19057 10069 
390 611
4489 16962 6202 18075 12428 16809 22261 23766 23255 20245 4109 31424 24008 1914 7536 16412 4447 31980 15031 25777 20128 25452 20140 27571 9529 32770 28169 11136 5221 3596 31619 32141 19216 3797 2311 10911 13116 14185 24132 26430 19619 28840 18537 29798 6814 7578 5936 14465 10565 7170 27419 25746 28120 13110 14589 31155 1630 25901 9917 17089 7307 21978 9559 5718 28647 15544 27683 29943 19245 5287 14682 22415 26865 32246 31702 16924 2950 1729 18969 28329 19487 19837 19089 9340 25908 12753 28955 3984 31784 1959 13765 11334 11816 132 10456 6999 25686 16148 2070 29197 13095 18698 14656 23219 18697 19239 19931 6474 12362 25495 29748 18557 26677 26244 31138 25303 6526 20710 16907 27700 27104 23960 22697 20341 13060 28849 32270 17528 8357 19729 22374 21064 31218 18490 8957 16784 6707 28770 5633 4950 23728 22726 18312 20453 32718 32832 19668 16907 17276 1564 27350 29317 18125 8488 242 30648 26854 3338 4354 28066 12571 25046 2205 10767 597 2528 12185 272 7607 18895 24526 11689 14465 3827 27165 13002 10548 30681 19634 18560 28663 19228 7813 10605 7737 20605 21543 21622 24984 16438 16827 5819 13825 15500 32322 29132 793 2468 3101 12826 10460 30615 27340 18859 16260 13000 30204 17170 3757 28278 13761 24735 32694 7565 1056 15315 3946 11714 11216 30998 736 30942 25665 25516 25503 24666 20190 1510 1798 18221 22928 4501 9452 24034 10883 11460 16771 29933 31020 30375 28290 18751 30859 22734 10115 32536 13997 30878 19881 16157 5494 23810 1671 10666 16541 7262 573 5225 17656 9925 27876 17956 11514 107 3885 9431 18042 93 25156 2570 30133 26479 6907 12637 14743 25462 28708 30053 17718 21176 8254 20150 21152 23457 31317 18753 4822 11448 28195 25569 24344 30929 26491 3632 3939 899 30952 14495 6713 20956 6545 10950 24992 21332 31327 5112 20845 5888 13922 2587 12994 24840 25201 27318 22742 13403 16401 21068 2804 9377 6041 7657 20423 1138 32147 11002 32212 17349 1892 18785 13640 4779 13104 5065 9972 7229 1784 8759 19606 28114 11013 19170 30298 24182 906 25925 20896 2712 3461 17266 19712 14339 742 21929 32139 13840 2547 11305 26964 3609 29675 19270 23344 6192 20201 1468 9145 18449 20236 20803 27545 18006 21205 18788 31493 10730 1744 21642 6791 26048 27767 8028 28929 16069 18515 25162 751 24083 15204 9574 
237 36
21351 3011 30326 888 7125 24710 17735 10744 186 29924 23108 8958 25331 1690 6982 8139 17996 908 4782 17682 12603 15024 27826 767 13519 5160 24308 11896 12515 5294 11865 16291 10391 12341 18200 22952 15105 12237 6929 32696 5151 11976 14945 6990 27152 29467 23091 14260 23172 9479 15832 5834 15474 10744 13851 771 14857 9901 10958 688 18152 27801 25257 20609 18065 22153 32085 15688 30491 22664 21443 24321 29736 18175 12994 12560 26135 6433 9169 2505 12667 10886 14385 2357 15051 16406 2623 25875 4699 21545 26881 13854 7306 28141 25087 10894 2621 2055 7508 16865 13709 19069 16388 30171 14013 7123 18576 29900 17775 19828 6600 20753 14871 26452 10209 18909 17327 22228 31687 4410 9645 1254 180 13130 4658 6001 20482 723 29828 25079 28478 12026 16318 18805 13437 26191 5574 8074 17651 11018 23168 14556 25025 15508 1857 17080 30479 14922 21127 7160 8539 25469 32743 13610 7440 6333 27718 15342 21595 599 10828 5536 32570 32796 17174 27959 2518 18020 28459 22637 13586 19539 23474 22561 9572 28507 20450 14329 21476 28674 18105 4427 5538 11238 20721 30014 11581 14590 21296 556 6833 17954 29823 26699 7537 25806 20733 8589 7350 8227 17620 24772 27158 1326 25453 31754 4696 109 28384 22112 12423 13455 25519 3221 30533 11900 27827 26986 25698 6805 21334 25843 21165 24381 16387 13120 24074 16536 24283 9789 19909 29074 15530 26327 31014 13860 28697 
246 760
23110 31443 25381 16693 17184 1401 20687 14238 26859 9031 19221 18933 14760 30731 23842 13067 12279 21167 7618 25543 6446 11010 17944 3856 428 11452 8339 11501 15211 29628 25630 14571 13013 23955 20375 12224 1497 5190 6531 4141 4413 13904 2048 25385 26678 16359 17658 3899 27190 6528 17058 31528 27397 28565 17436 17358 8467 14246 24184 2384 19981 1662 1508 6003 21934 7095 31520 27976 14040 18048 11570 26636 6883 1950 25434 5452 28469 5297 17224 28748 15634 8377 1933 3754 13238 3544 31312 32831 6305 27738 2114 26182 4659 27378 32724 1472 120 7238 20647 22807 22597 13742 24387 25857 13444 11065 16653 21671 4556 19698 29811 8455 26677 18785 26986 18326 5994 14676 28186 21546 22783 6367 21463 16920 11525 20897 24660 30873 29916 14921 24339 26227 12361 12121 26400 213 4296 10538 10495 10626 22308 29488 22755 16433 12100 26752 2337 25396 32328 22927 2275 9049 22229 32573 6945 12046 1964 7954 6605 14536 30850 17607 22862 7333 18700 13576 24624 2384 26201 32120 18202 12325 2660 17910 21760 16533 23018 18207 11161 13004 5241 2579 6164 9618 2762 21884 23559 21359 20304 27597 680 25363 21777 32456 5695 5876 28853 31811 18907 8249 28636 6187 27460 6359 15029 14281 12594 26874 13261 8772 14780 7078 18980 6760 19731 26270 29062 17829 14448 27013 29976 1115 21835 26691 22578 8021 4302 1194 14700 28307 1929 15194 27644 23884 2407 10209 15167 28467 6625 21543 3221 11002 16497 5898 21362 3604 
437 666
17989 2994 28096 30278 16278 11821 26947 12320 3360 21388 20852 26595 10510 31067 10622 24738 9337 7519 9442 14698 13707 9001 18107 8053 28901 14315 1606 28832 12863 10379 7329 25119 30798 24054 24531 32624 14315 29623 20237 5517 27249 7368 13360 24166 23399 19111 32398 142 31071 19117 15093 14133 11805 19868 19690 13910 508 3447 19762 7985 3726 1430 12695 6396 1155 28921 8680 25046 18636 733 7746 27644 27904 3469 23182 23359 1790 2939 29515 30390 11220 16015 24462 25264 4575 22975 17478 14060 7732 26848 3269 17677 24895 27323 27772 9032 18090 20836 11782 20320 31872 23751 29545 32212 21907 900 19352 26967 11035 24061 16700 13952 415 18348 18099 16323 29977 10939 9049 23968 12373 10702 17927 32538 29594 7204 8410 9835 28300 4062 29238 7670 29009 18078 7946 31101 14579 24660 597 14228 23301 10325 21993 5973 5258 14510 6255 30294 32786 26551 5291 19160 30502 13291 28161 3649 8934 8694 22072 11213 32753 16656 20607 18304 22917 18826 9645 29023 32744 2470 28109 6367 15878 5292 21210 21662 21083 26682 28871 21073 28435 31708 30402 765 1392 28380 28090 22379 24685 18524 18463 8406 21153 32161 7913 27110 31589 12978 16347 23522 26425 26914 28937 1057 3940 4394 3480 3521 31963 12539 24517 17308 4708 5713 23592 28280 16891 17996 8762 25470 12661 8060 26134 22254 4707 25146 10108 18192 12601 4927 23067 4974 32688 27759 25360 5184 7784 20470 4010 17960 10706 29078 26992 24095 20045 10815 2917 22959 26511 10721 23690 24420 12103 31614 21413 893 16128 3971 10305 17638 26953 16472 5366 22123 11795 1334 3818 2086 20058 18233 5784 25913 17160 3254 1488 22979 6934 28366 32511 29964 1268 3395 25009 7171 1835 29835 26708 16363 11329 10787 21311 27985 9692 21286 17651 18241 11051 10526 13216 17081 5762 12570 5099 13918 1604 26881 8430 4718 1286 24357 25984 21750 31985 26651 27319 3653 31336 26315 12204 2464 491 23235 20094 28215 32060 10449 29338 27736 960 20944 1081 19171 551 3010 11562 26878 1360 28959 6865 2057 11366 14957 8560 29141 1616 13841 9381 20240 24136 29285 26932 16900 15305 19978 13319 6530 6575 25216 23155 12312 6298 31987 10173 19338 12435 6488 15325 4224 9714 5049 18305 24189 17790 973 18773 10256 11223 30412 12418 6990 7185 31617 2164 11903 10658 13473 4577 25880 14718 25947 27911 24810 7808 11312 14645 5016 21772 24581 19852 9294 8691 17271 28423 23984 6294 622 10169 14353 19535 11765 9046 21478 12556 10825 13557 30842 28477 17199 27156 27363 12017 23179 20189 12596 7739 25903 18344 5620 14043 10737 9152 32719 1818 30464 19818 29068 502 
216 736
15139 17983 6783 25395 1578 32192 11905 27363 19006 11611 1123 12893 12969 26825 24655 12088 22653 14175 29844 12695 22046 27014 12616 1347 13312 25195 14129 10493 4786 10744 29158 4114 13852 23298 7473 8751 22021 16836 14776 28023 22488 16012 13532 16083 21927 16564 18402 5257 5744 28116 21072 7953 23455 18742 13210 17444 10197 28554 26622 22549 25320 15067 27821 356 32555 25282 12075 31377 1007 27866 3064 9774 8521 371 8304 10552 12566 30859 29680 32679 18749 13050 15418 16196 17204 5100 16667 31931 23750 11924 9035 1777 22035 31775 2708 27799 12504 11751 32566 32768 29936 28650 29323 19759 7707 12154 15079 13877 5887 14902 1730 10682 25964 18744 2841 10201 27620 19600 28945 2353 25691 2801 18142 15416 30779 27051 30691 27859 6571 28889 4122 18980 529 24540 20151 24227 31189 26120 19464 11889 19817 21404 13984 4073 4998 22264 27572 29303 10424 6666 28357 1965 1052 21514 1151 659 18792 9063 29922 24895 15405 17409 14495 29862 30809 30104 5573 6834 5120 9860 28572 24213 3676 13991 19921 29532 12306 26273 24276 15559 16566 18242 31103 23128 13714 15685 25080 26794 15779 27963 21976 27908 20197 9157 9101 7982 6353 3590 20733 1411 32803 22104 5483 5960 2016 31038 10911 22844 19352 26267 14657 10352 28392 17185 25735 10073 
49 772
24075 14608 292 15603 12047 783 30537 15451 28814 20395 28003 6271 10767 7590 26431 1478 16107 317 11571 23550 21833 16208 2737 10912 10573 9155 23323 25174 8333 5924 22745 14266 12741 14222 25258 9057 29899 30925 11063 8984 28357 4034 1774 8301 28313 11529 19469 8306 3493 
338 138
30973 8381 10417 4653 5346 17843 25889 6946 32368 15879 5743 3231 7424 32637 6105 24379 10211 16628 28247 18267 30710 21923 18128 2714 6006 19875 9357 17519 1046 32685 21486 24074 22351 24614 30044 15742 18680 18776 13438 28709 16406 6403 32688 21568 26101 18196 21425 1503 28536 22964 28571 4865 4269 23172 1305 15099 20132 12583 14548 15646 9090 1794 1838 31380 10449 6207 24883 28555 15386 5656 5580 18326 20918 32338 23350 4431 27105 3079 3251 21826 23568 9641 28800 5126 27781 27716 6662 28589 24608 20483 15444 1119 632 19958 205 13377 21034 15122 13854 27291 18289 16506 27508 6715 26871 27901 19435 31508 8939 32735 20041 19243 5204 20700 3308 5812 18085 31082 23041 19319 913 9023 25591 7033 15648 8211 32307 29401 15363 11943 25094 1272 2085 4358 31708 13532 16258 22538 6326 23767 20284 2083 2010 14691 17131 31572 1224 31569 7733 9342 32113 17699 29856 6113 2212 6070 13219 31968 8393 29220 21251 19716 30417 10472 24413 1092 29037 14403 4862 9860 11390 5599 28923 15963 3277 17843 5759 14617 15357 13862 13919 17417 8382 10103 29428 3827 10051 8796 2601 6117 28149 16873 2003 25651 10857 21684 29249 19133 23183 6455 9492 13168 5677 6517 20049 28688 5260 27105 16821 4660 14962 20347 5705 25234 17487 4412 16950 23932 24763 30900 10457 22699 20256 17906 20250 13884 22196 6820 10746 18680 10667 8319 15212 19928 2218 28385 29030 7880 14974 29621 31543 684 11278 25496 2331 16033 19598 30009 10765 19589 31222 27654 15324 23225 4531 13829 8577 31013 31132 8447 13232 6013 13607 27364 8109 14531 23261 12651 10337 32782 32267 1273 32173 31369 24763 8900 7678 19134 482 6712 30242 6421 23515 24986 13146 28055 26388 7659 31664 22264 13576 8719 138 725 14189 26926 8064 11066 1966 4187 1387 32103 17051 4979 1982 24020 19552 20808 10266 11634 4275 6911 30236 29748 4946 21175 17274 5326 25773 23916 24142 28093 20191 27674 17841 2951 20195 24613 3212 20548 11781 20216 24235 13405 20374 26860 23571 1172 
425 740
18284 16022 16708 2410 26200 32164 746 20803 19622 25700 2021 997 10319 19971 20346 12555 24937 31821 26139 32243 26768 15482 18153 4298 17547 20042 30327 20581 9005 16724 15057 19505 29048 15514 5309 24685 16137 25698 24128 27286 28848 21057 12900 8739 1630 21369 569 26995 30788 24965 31514 11012 28473 2289 8753 5861 23644 12911 18750 21614 13307 552 16531 31790 20404 16720 27235 28197 15322 1533 4925 22579 29723 3385 13268 209 21343 31573 32769 10751 15090 30337 26379 8317 13542 20378 26917 19262 5555 11587 26168 20250 24865 27959 18915 6851 30312 12254 14529 28410 18325 16926 8938 14083 8239 22899 24508 17750 16155 22082 14043 18583 18365 106 31350 3272 20064 29213 5836 23742 984 9454 15087 6097 17509 21802 32456 20440 3547 12140 10844 853 8448 31852 589 18412 21046 16152 15780 31002 19802 30950 2253 16163 20715 2152 14819 588 9327 22049 6703 7755 19986 17303 26859 27132 19345 4265 31563 26453 21078 11043 7614 12188 23269 7312 6769 28631 31606 2639 2324 2437 21987 31721 19426 11147 7365 23458 24027 30577 29658 25934 22100 1742 12203 30566 17842 25857 14443 31323 20339 20770 5153 21034 9582 23609 23562 3633 18523 14253 8504 19265 12693 18431 1152 31732 18975 20407 1427 2881 21221 18079 19819 3405 8773 12592 30697 29316 23246 3146 8380 28905 32584 8488 14212 6058 2036 21906 24375 16632 23829 19540 23748 22037 23214 27120 22775 6575 16782 4496 31133 27756 14730 11838 5608 16326 18698 23639 2581 11033 26055 18237 28235 22266 722 3917 13511 13003 16345 19473 29494 10380 7590 32438 23394 17140 23528 2240 10001 31169 25654 31484 31530 6220 26193 20662 1562 29280 2473 27437 23926 23372 22563 24900 24290 4919 17946 30586 6193 7860 31652 21532 31438 15511 17174 17338 24154 28166 16586 8663 5782 17619 22559 14820 8324 27657 25026 13424 10953 7876 5932 25753 32086 326 22380 2502 1095 4168 10535 22936 523 6798 7645 26141 11462 30910 31341 13383 15091 30578 18652 5789 9038 10124 14162 8241 5105 2250 26725 17362 25789 17050 6298 29570 2299 24367 15657 21547 17475 15572 32769 22925 9885 10624 15628 24187 5095 23723 25780 3399 23159 17249 26534 11387 7425 21141 4054 9161 5050 21431 27331 19781 26813 24457 10492 26322 13230 31534 5859 14298 22686 2947 4785 16195 20015 4787 9146 32158 3833 27654 5875 12193 14892 31226 17585 3287 10102 31381 25523 24214 29223 30201 3427 818 16275 31545 22361 12656 17591 8043 9761 4157 20256 29439 7486 30368 21769 24911 20590 26606 20676 11894 6704 8253 27483 
