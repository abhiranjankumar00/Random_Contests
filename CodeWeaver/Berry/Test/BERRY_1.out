40 806
304 64604
131 4397
363 20160
120 2798
219 17368
403 86387
119 3418
382 39913
98 5906
389 10764
236 107324
244 5224
435 10866
215 5038
47 912
337 40072
423 9826